test/testdrive/primary-key-optimizations.td

```diff
@@ -57,89 +57,89 @@ $ kafka-ingest format=avro topic=t1 key-format=avro key-schema=${keyschema-2keys
 
 # Optimization is possible - no distinct is mentioned in the plan
 
-> EXPLAIN SELECT DISTINCT key1, key2 FROM t1;
+> EXPLAIN WITH(no_notices) SELECT DISTINCT key1, key2 FROM t1;
 "Explained Query (fast path):  Project (#0, #1)    ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT DISTINCT key2, key1 FROM t1;
+> EXPLAIN WITH(no_notices) SELECT DISTINCT key2, key1 FROM t1;
 "Explained Query (fast path):  Project (#1, #0)    ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT DISTINCT key2, key1, key2 FROM t1;
+> EXPLAIN WITH(no_notices) SELECT DISTINCT key2, key1, key2 FROM t1;
 "Explained Query (fast path):  Project (#1, #0, #1)    ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT key2, key1 FROM t1 GROUP BY key1, key2;
+> EXPLAIN WITH(no_notices) SELECT key2, key1 FROM t1 GROUP BY key1, key2;
 "Explained Query (fast path):  Project (#1, #0)    ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT key2, key1 FROM t1 GROUP BY key1, key2, key2 || 'a';
+> EXPLAIN WITH(no_notices) SELECT key2, key1 FROM t1 GROUP BY key1, key2, key2 || 'a';
 "Explained Query (fast path):  Project (#1, #0)    ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT DISTINCT key1, key2, nokey FROM t1;
+> EXPLAIN WITH(no_notices) SELECT DISTINCT key1, key2, nokey FROM t1;
 "Explained Query (fast path):  ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT key1, key2, nokey FROM t1 GROUP BY key1, key2, nokey;
+> EXPLAIN WITH(no_notices) SELECT key1, key2, nokey FROM t1 GROUP BY key1, key2, nokey;
 "Explained Query (fast path):  ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
 
-> EXPLAIN SELECT key1, key2 FROM t1 GROUP BY key1, key2 HAVING key1 = 'a';
-"Explained Query (fast path):  Project (#3, #1)    Filter (#0 = \"a\")      Map (\"a\")        ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)Notices:  - Notice: Index t1_primary_idx on t1(#0{key1}, #1{key2}) is too wide to use for literal equalities `#0{key1} = \"a\"`.    Hint: If your literal equalities filter out many rows, create an index whose key exactly matches your literal equalities: (#0{key1})."
+> EXPLAIN WITH(no_notices) SELECT key1, key2 FROM t1 GROUP BY key1, key2 HAVING key1 = 'a';
+"Explained Query (fast path):  Project (#3, #1)    Filter (#0 = \"a\")      Map (\"a\")        ReadIndex on=t1 t1_primary_idx=[*** full scan ***]Used Indexes:  - t1_primary_idx (*** full scan ***)"
```

Comment:
Good idea to introduce `WITH(no_notices)` for testdrive explains! I remember these were super annoying when I was initially adding the notices.